test/e2e/antctl_test.go

```diff
@@ -379,6 +380,7 @@ func createAntctlServiceAccount(t *testing.T, data *TestData, name string) {
 			Name:      name,
 		},
 	}
+	log.Infof("Creating ServiceAccount '%s/%s' for antctl test Pods", data.testNamespace, name)
```

Comment:
it's strange to add a logrus log message here. I know that logging in e2e tests could be improved / unified, but at the moment I would recommend moving this log message to the call site that interests you.

Reply:
Got it, removed.